src/envoy/http/jwt_auth/jwt_authenticator.cc

```diff
@@ -59,16 +59,6 @@ void JwtAuthenticator::Verify(HeaderMap& headers,
   headers_ = &headers;
   callback_ = callback;
 
-  // Sanitize the JWT verification result in the HTTP headers
-  for (const auto& rule : store_.config().rules()) {
-    if (!rule.forward_payload_header().empty()) {
-      ENVOY_LOG(debug, "Sanitize JWT authentication output header {}",
-                rule.forward_payload_header());
```

Comment:
This code should not be removed.

Reply:
For the same reason as above. We will remove this forward_payload_header in our API.